packages/web3-eth-contract/src/contract.ts

```diff
@@ -256,10 +263,9 @@ export class Contract<Abi extends ContractAbi>
 			// eslint-disable-next-line @typescript-eslint/no-explicit-any
 			method: ContractBoundMethod<any>;
 			// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I think this line is not needed anymore.

Reply:
yes. thanks. removed.